src/geom/elem_cutter.C

```diff
@@ -283,11 +283,11 @@ void ElemCutter::cut_2D (const Elem & elem,
   _inside_elem.clear();
   _outside_elem.clear();
 
-  for (const auto & elem : _inside_mesh_2D->element_ptr_range())
-    _inside_elem.push_back (elem);
```

Comment:
How did this *not* trigger -Wshadow before?

Reply:
Code is only active in `--disable-strict-lgpl` builds, not sure if we have one of those that *also* has `-Werror -Wshadow`